config/build/rollup.config.js

```diff
@@ -55,6 +55,12 @@ const createOutputConfig = (pkgName, format, targetFile) => ({
     sourcemap: true,
     format,
 
+    // These two settings are to keep the builds as similar to pre-Rollup v4 as
+    // possible until we get rid of CJS builds.
+    // See: https://rollupjs.org/migration/#changed-defaults
+    esModule: true,
+    interop: "compat",
```

Comment:
So we can get rid of this after https://github.com/Khan/perseus/pull/2331 lands?

Reply:
Yes, and you can verify its deleted in the PR here: https://github.com/Khan/perseus/pull/2331/files#diff-45f213140a165c743b42e38a00d97984fcec3f841195e8587d03a861d402ca17R63-R75